README.md

```diff
@@ -1,6 +1,7 @@
 # OpenSAFELY Airlock
 
-An in-development component of the OpenSAFELY research framework.
+A tool that allows researchers to request the release of a set of files from an OpenSAFELY backend, allows output checkers to review the contents of those files for disclosivity, and then releases those files deemed safe to job-server.
```

Comment:
```suggestion
A tool that allows researchers to view files in an OpenSAFELY backend and request their release. It then allows output checkers to review the contents of those files for disclosivity and discuss them with their colleagues. Finally it can release those files deemed safe to job-server.
```

Reply:
@inglesp better?

Reply:
too slow